python/ucxx/ucxx/_lib/arr.pyi

```diff
@@ -1,10 +1,12 @@
-# SPDX-FileCopyrightText: Copyright (c) 2022-2023, NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: Copyright (c) 2020-2024, NVIDIA CORPORATION & AFFILIATES.
 # SPDX-License-Identifier: BSD-3-Clause
 
-from typing import Tuple
+from typing import Generic, Tuple, TypeVar
 
-class Array:
-    def __init__(self, obj: object): ...
+T = TypeVar("T")
+
+class Array(Generic[T]):
```

Comment:
Same comment here as in ucx-py

Reply:
Same answer: https://github.com/rapidsai/ucx-py/pull/1080#discussion_r1783965174